.github/ISSUE_TEMPLATE/apri-una-segnalazione.it.yaml

```diff
@@ -0,0 +1,54 @@
+name: Apri una segnalazione - IT 🇮🇹
```

Comment:
Sorry if I'm being pedantic today, but flags represent countries, not languages, so I'd rather not use the emoji even if it's visually appealing.